lib/msf/ui/console/command_dispatcher/common.rb

```diff
@@ -158,6 +158,31 @@ def index_from_list(list, index, &block)
     yield list[idx]
   end
 
+  # Trims starting `.`, `./` `/`, `+path_head+/`, & `/+path_head+/` from +path+. Also trims trailing `.+extension+`
+  # from +path+, and any possible combination of misspellings of +extension+.
+  #
+  # @param path [String] The path to be trimmed
+  # @param path_head [String] The top-level directory that should be removed from the path
+  # @param extension [String] File extension to be trimmed from +path+. `.` is automatically included. Defaults to `rb`.
+  # @return [String] Altered +path+. Will return unaltered +path+ if regex constructed with +path_head+ & +path+ is not detected
+  def trim_path(path, path_head, extension="rb")
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#spaces-around-equals

If you're coming from Python, I can understand this! Don't feel the need to change it, necessarily.

Reply:
Looks good 👍  I think operators are nicer due to the extra clarity anyway, and I only used the `foo=blah` syntax in commandline args so this is a two birds one stone situation